### ISSUE TITLE
Normalizing to 100,000 Population

### ISSUE THREAD

**Issue by DTPOTO** [nychealth/coronavirus-data#10]
I am concerned that the process of normalizing the data to represent 100,000 population may have been based on national coefficients and not NYC based coefficients.    I am specifically concerned that the AGE-GROUP presentation [here](https://github.com/DTPOTO/DTPOTO_Repository/blob/master/COVID-19%20NYC%20Health%20Prevalence%20by%20Age-Group.pdf) is showing that AGE does not really matter.  

When I used Coefficients based on NYC demographics I came up with a different story.  I have been especially concerned with the 65-74 age group (Young Retired) have had a disproportional high level of disease incident level from the onset.
Please see my further explanation & assessment [here](https://github.com/DTPOTO/DTPOTO_Repository/blob/master/NYC_Incident_level_by_Age_Group.pdf)

The readme documentation points to CDC with regards to 100,000 standardization.   My review of the CDC document talks about National Coefficients based on National Demographics for a National Survey.

**Comment by ret394** [comment]
went to https://github.com/DTPOTO to follow your account. hope to learn a lot.

can anyone or you, sir, please convert https://github.com/DTPOTO/DTPOTO_Repository/blob/master/NYC.Cases_Adj1.xlsx to an open format and provide it in your repo. if it contains data not in other files in repo. pdfs are better as major web browsers can open them.

https://www.openoffice.org/why/why_odf.html#what-are-odf-documents . 

If you want to try the software, 2 choices I found are. Either on cheaper machine or in a VM (needs another software)
https://www.openoffice.org/download/index.html
https://www.libreoffice.org/download/download/

Portable version https://www.libreoffice.org/download/portable-versions/

Do check out some tutorial articles or videos for the install and for using them. 
A tip is don't install extensions unless needed, just like for web browser extensions.

check the source code 
- OpenOffice https://openoffice.apache.org/downloads.html

And if you like it, please share and ask other people to try it as success depends on market share and feedback more than quality of software.

**Comment by DTPOTO** [comment]
I will be closing this issue because the issue seems to be on my part.   I have pulled the Census data for ACS 2018 1yr county level CBSA level data to recalculate the stratum coefficients.   I was able to exactly match the NYC Dept of Health numbers.  

I am including 2 graphs to show the grow and the declining rate of increase by Age-Group.
![image](https://user-images.githubusercontent.com/63018432/78594708-c5b7eb80-7816-11ea-88e7-0c670859e4e7.png)
 
As of the the start of April, the older one groups are showing a greater rate of increase than younger age group 45-64..  

This chart shows the Rate of Change.  
![image](https://user-images.githubusercontent.com/63018432/78595081-60b0c580-7817-11ea-85b6-9263b3d4e513.png)

This chart showing that the "Rate-of-Increase" relative to the prior days New-Case has been decreasing across all age groups.   Notably the Rate-of-Increase for the older Age Groups have been declining slower than the younger groups.   This is another way to expressing the prior graph but is actually the same story.

When the Rate-of-Increase reaches ZERO then we are technically at the Daily-NEW-Case APEX.  Of course that just means we getting to the APEX of Total-Cases.  The Apex of Total Cases can be calculated one we have stop the increase of New cases.

The declining Rate-of-Increase sounds like an oxymoron!   This is like trying to understanding the slope the line as it changes when you get to the Apex of a polynomial graph.    This supports  what Gov Cuomo has been claiming for days with regards to reaching an apex in the near future.
